config/config.schema.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/ternaustralia/rs-remote-job/config.schema.json",
+  "title": "Endpoints",
+  "description": "Config json for resource server to define endpoints",
+  "type": "object",
+  "properties": {
+    "parameters": {
+      "description": "List of parameters supplied to this resource server",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "type": {
+            "oneOf": [
+              {
+                "type": "string"
+              },
+              {
+                "type": "number"
+              }
+            ]
+          },
+          "default": {
+            "type": "string"
+          }
+        },
+        "required": [
+          "name",
+          "type",
+          "default"
+        ]
+      },
+      "minItems": 0,
+      "uniqueItems": true
+    },
+    "endpoints": {
+      "description": "List of endpoints to be defined",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "name of this endpoint"
+          },
+          "async": {
+            "type": "boolean"
+          },
+          "httpMethod": {
+            "type": "string",
+            "enum": [
+              "GET",
+              "POST",
+              "DELETE"
+            ]
+          },
+          "exec": {
+            "type": "object",
+            "description": "The command to be executed in login host. Note that the paramters defined in {} needs to be substitued with paramers supplied to this command, and server",
```

Comment:
- paramters -> parameters
- substitued -> substituted
- paramers -> parameters

Reply:
lol thank you but that wasn't written by me hahahah I and didn't check